internal/cli/cli.go

```diff
@@ -49,6 +49,7 @@ func (c *CLI) Run(ctx context.Context, args ...string) error {
 		cmd := create.New(cmd, c.in)
 		cli := cli.Command("create", "create a new app")
 		cli.Arg("dir").String(&cmd.Dir)
+		cli.Flag("css", "add a css").String(&cmd.Css).Default("normalize")
```

Comment:
Let's remove this flag for now, I'm not ready to expose it just yet, especially since it's not really possible to select an alternative.

Reply:
Done